README.md

```diff
@@ -2,16 +2,26 @@
 
 A library of OpenAssetIO extensions for use in Media Creation workflows.
 
-> Note: This repository is currently in a pre-alpha state, and so should
+> **Warning:**
+> This repository is currently in a beta state, and so should
 > not be used for any production critical applications.
 
 Included are several well-known Traits and Specifications for use in
-OpenAssetIO hosts and managers. For more information on this mechanism,
-see the [OpenAssetIO docs](https://openassetio.github.io/OpenAssetIO).
-
-MediaCreation is an automatically generated Python package,
+Digital Content Creation tools and wth Asset Management Systems found in
+the media production space. For more information on the Traits and
+Specifications mechanism, see the [OpenAssetIO
+docs](https://openassetio.github.io/OpenAssetIO).
+
+The project aims to align with the [MovieLabs
+Ontology](https://movielabs.com/production-technology/ontology-for-media-creation/)
+project wherever possible. Presently this equates to traits and their
+properties directly mapping to specific ontology sub-schemas and their
+property definitions.
```

Comment:
[consider][minor]
```suggestion
The project aims to eventually align with the [MovieLabs
Ontology](https://movielabs.com/production-technology/ontology-for-media-creation/)
project wherever possible. Presently this equates to traits and their
properties directly mapping to specific ontology sub-schemas and their
property definitions. 

> **Note** 
> The ontology is still a work in progress,
> so alignment is limited at this time
```
or something to that effect.  Otherwise it gives the impression that the traits we have already align with the ontology.

Reply:
Hows [this](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/71/commits/e398e0d62bd2875323c58ce23204f76e49b94cd3)